pyoracc/test/atf/test_atflexer.py

```diff
@@ -896,6 +909,40 @@ def test_note_ended_by_strucuture(self):
             ["REVERSE"]
         )
 
+    def compare_note_ended_by_line(self, line_label):
+        'Helper for Note para state termination.'
+        # Sample text.
+        line1 = u"a-šar _saḫar.ḫi.a_ bu-bu-su-nu"
+        line2 = u"a-kal-ši-na ṭi-id-di"
+        # Generate the successive line numbers in the same style.
+        label1 = line_label
+        next = int(label1[:1]) + 1
```

Comment:
Very minor, but this shadows the built-in `next` function. Perhaps rename to `next_label`?

Reply:
Ok.